### ISSUE TITLE
Config.json is not stored in .keeper folder unless -lwsc flag

### ISSUE THREAD

**Issue by mxmorin** [Keeper-Security/Commander#810]
According to the documentation, default location of config.json should be $HOME/.keeper folder. This is not the case (created in $HOME) unless the --lwsc flag is set

```
    elif opts.launched_with_shortcut:
        launcher_keeper_folder_path = Path.home().joinpath('.keeper')
        launcher_keeper_folder_path.mkdir(parents=True, exist_ok=True)
```

Thus, --config flag shoud override KEEPER_CONFIG_FILE env variable

**Comment by groner** [comment]
It accesses `config.json` in whatever directory you start it in. Not a very helpful behavior, IMO.

**Comment by ga-tb** [comment]
I guess this part of the documentation is relevant:

> If you use Commander from a PIP3 or source installation, the configuration file will be created in the current folder where the Commander executable is started from.

**Comment by sk-keeper** [comment]
Yes, this part is correct. `keeper` command looks current working directory for `config.json` file if `--config` option is missing

**Comment by dg-nvm** [comment]
why exactly default has to be workingdir? On linux (or WSL) there is no way of installing a "binary release" that will obey XDG. This is raising a risk of using keeper to do some work on project and accidentally commiting config.json using `git add --all`

**Comment by dg-nvm** [comment]
@sk-keeper I understand but this is additionall user action that if not done, impacts security. Config file includes sensitive data like device_token, private_key and clone_code. [LastPass CLI](https://github.com/lastpass/lastpass-cli/blob/a5b93b9015e1524f557e3eb4fdf688a407d41c73/config.c#L77), [Bitwarden](https://github.com/bitwarden/clients/blob/3da7fc7cb39f575a96c52a6a17a4eba8cf651093/apps/cli/src/bw.ts#L151) and [1Password](https://developer.1password.com/docs/cli/config-directories)  obeys [XDG Base Directory standard](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) .

Why Keeper is different?

**Comment by groner** [comment]
I have to agree with @dg-nvm, the existing behavior of keeper commander is not desirable.

**Comment by sk-keeper** [comment]
I agree it makes sense to store config file in a predefined location if it is missing in the working dictionary.
It is a safe change

**Comment by sk-keeper** [comment]
fixed in 16.9.3
